pkg/util/cluster/cluster.go

```diff
@@ -357,12 +391,12 @@ func (c *Cluster) createCluster(ctx context.Context, vnetResourceGroup, clusterN
 				},
 			},
 			APIServerProfile: api.APIServerProfile{
-				Visibility: api.VisibilityPublic,
+				Visibility: api.VisibilityPrivate,
```

Comment:
let's make this change dependent on CI for now and poll the team for what to do about it